### PR TITLE
fix(monitor install): Fix ubuntu monitor installation prerequisites

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
Since python 3.6 installation is not required if the machine uses ubuntu 20,
I seperated the prerequisites installation script of older versions
(ubuntu 16 and 18) between the newer ones (20 and forwards)

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fruch — DISMISSED** [review]
LGTM

File: sdcm/cluster.py [hunk 1]
```diff
@@ -4989,7 +4989,7 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
                 sh get-docker.sh
                 systemctl start docker
             """)
-        elif node.is_ubuntu():
+        elif node.distro.is_ubuntu18 or node.distro.is_ubuntu16:
```
Comment (fgelcer) [inline sdcm/cluster.py:4992]: ubuntu 16 is deprecated and not supported anymore (only on old branches, and i expect this commit will never get to these branches, please remove it

File: sdcm/cluster.py [hunk 2]
```diff
@@ -5006,6 +5006,20 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
                 pip3 install -I -U psutil
                 systemctl start docker
             """)
+        elif node.is_ubuntu():
```
Comment (fgelcer) [inline sdcm/cluster.py:5009]: so here we will have Ubuntu 20.04 and Ubuntu 22.04... why not to unify all the 3 supported Ubuntu versions?

**Review by fgelcer — CHANGES_REQUESTED** [review]

File: sdcm/cluster.py (hunk 2 above)
Comment (rayakurl) [reply sdcm/cluster.py:5009]: Also, as we discussed face to face, please add the if statement of ubuntu 18 into is_ubunto and add a TODO comment to remove it once the we stop supporting Ubuntu 18

**Review by rayakurl — APPROVED** [review]
Approved from my side. Just make sure it works both on ubuntu 18 & 20

**Comment by ShlomiBalalis** [comment]
Tested, works with both

File: sdcm/cluster.py [hunk 3]
```diff
@@ -4996,16 +4996,18 @@ def install_scylla_monitoring_prereqs(node):  # pylint: disable=invalid-name
...
-                python3.6 -m pip install pyyaml
-                pip3 install -I -U psutil
+                apt-get install -y python3-pip
+                python3 -m pip install pyyaml
+                python3 -m pip install -I -U psutil
                 systemctl start docker
             """)
+            if node.distro.is_ubuntu18:
+                prereqs_script += dedent("""
+                    add-apt-repository -y ppa:deadsnakes/ppa
```
Comment (fruch) [inline sdcm/cluster.py:5008]: now you are not using the version out of deadsnake at all, 
i.e. this part can be remove and everything would work

the whole idea of use deadsnakes repo, was to install python 3.6, which wasn't available at some point, if it's available and everything works on ubuntu 18, no point of adding that repo at all.

Comment (ShlomiBalalis) [reply sdcm/cluster.py:5008]: Then I guess it's just unnecessary completely. I'll remove it.

File: sdcm/cluster.py (hunk 1 above)
Comment (fgelcer) [reply sdcm/cluster.py]: @rayakurl , it is still supported to the "old" released versions of Scylla, but we dropped the support on newer versions

**Comment by ShlomiBalalis** [comment]
@fruch Can you backport it to manager 2.6 as well?

**Pull request merged**